redwood-treehouse/src/hostMain/kotlin/app/cash/redwood/treehouse/TreehouseLauncher.kt

```diff
@@ -37,6 +37,7 @@ import okio.Path
 public class TreehouseLauncher internal constructor(
   private val platform: TreehousePlatform,
   public val dispatchers: TreehouseDispatchers,
+  private val eventListener: EventListener,
```

Comment:
For follow-up: actually invoke methods on these things.

Reply:
We should also add docs for the listener methods to the follow up 🙏